eoc_journal/eoc_journal.py

```diff
@@ -99,12 +99,36 @@ class EOCJournalXBlock(StudioEditableXBlockMixin, XBlock):
         scope=Scope.content,
     )
 
+    should_metrics_section_be_displayed = Boolean(
```

Comment:
Nit: I would prefer if these were named `display_...` instead of `should_..._be_displayed` (ie. `display_metrics_section` instead of `should_metrics_section_be_displayed`) because its shorter and I think more consistent with how we usually name boolean fields. It's purely a matter of preference though, so if you'd prefer to keep existing names that's totally fine with me :)

Reply:
👍